site/docs/v1/tech/admin-guide/releases.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: doc
+title: Releases
+description: How can you see when a new version of FusionAuth is released?
+navcategory: admin
+---
+
+== Overview
+
+FusionAuth is a single tenant SaaS or downloadable solution. This fact means that you control when upgrades occur, rather than having an upgrade with new functionality and possibly breaking changes forced on you and your team.
+
+[NOTE.note]
+====
+Authentication and authorization are core dependencies of your application, and we hate forced dependency upgrades.
+
+You and you alone control when you upgrade.
+====
+
+* <<Release Notifications>>
+* <<Early Access Program>>
+
+== Release Notifications
+
+include::docs/v1/tech/shared/_release-notification.adoc[]
+
+== Early Access Program
+
+[NOTE.since]
+====
+Available since 1.41.0
+====
+
+FusionAuth has an early access program (EAP).
+
+The EAP allows you, within certain constraints, to test drive the features and functionality of the next release of FusionAuth. 
```

Comment:
Another component of the EAP is that any new features that will require a license at GA - will not require a license during EAP to allow for everyone to test.